DEVDOC.md

```diff
@@ -49,8 +49,11 @@ mkdir <path-outside-agi-source>/agi-gofuse
 # Run gofuse with the previous directory as a target
 bazel run //cmd/gofuse -- -dir <path-to-agi-gofuse>
 
-# If you build with bazel build -c dbg pkg, the <path-to-bazelout> is `k8-dbg` on Linux.
-bazel run //cmd/gofuse -- -dir <path-to-agi-gofuse> -bazelout <path-to-bazelout>
+# If the previous command fails to correctly guess the sub-directory under `bazel-out`,
```

Comment:
Does "previous command" here refer to line 50? I see this isn't just a comment update but a modification of the command. I have to assume previous command does not refer to the line being replaced as the reader won't be able to see that anymore after submission.

Reply:
Correct, "previous command" refers to line 50.